### PR TITLE
Add C declarations needed for zmq http server

### PR DESCRIPTION
**Pull request by bneo99**
Required by [koreader to create a zmq http server](https://github.com/koreader/koreader/pull/10223/)

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader-base/1595)

### CODE REVIEW

File: ffi/zeromq_h.lua [hunk 1]
```diff
@@ -109,4 +113,6 @@ int zpoller_add(zpoller_t *, void *);
 void *zpoller_wait(zpoller_t *, int);
 bool zpoller_expired(zpoller_t *);
 bool zpoller_terminated(zpoller_t *);
+int zmq_send(void *, void *, size_t, int);
+int zframe_send(zframe_t **, void *, int);
```
Comment (Frenzie) [inline ffi/zeromq_h.lua:117]: Not the biggest deal but these seem to be in the "wrong" location compared to their definition?

**Review by Frenzie — APPROVED** [review]
@NiLuJe ?

**Comment by NiLuJe** [comment]
LGTM besides the ordering nit mentioned above (which will make diff'inf further updates confusing, so, yeah, please fix it ;)).

**Comment by bneo99** [comment]
Whoops, should be fixed now

File: ffi-cdecl/zeromq_cdecl.c [hunk 2]
```diff
@@ -112,7 +119,6 @@ cdecl_func(zhash_cursor)
 cdecl_func(zhash_lookup)
 cdecl_func(zmsg_destroy)
 cdecl_func(zhash_destroy)
-cdecl_func(zframe_destroy)
```
Comment (Frenzie) [inline ffi-cdecl/zeromq_cdecl.c:115]: It looks like all the destroys are purposefully together. You prefer them as all the zframes together?

Comment (bneo99) [reply ffi-cdecl/zeromq_cdecl.c:115]: There is other destroys that are grouped with their own class functions. I'll move the other destroys to their own  groups. So all functions to be grouped by their class (zhash, zctx, zframe etc.)

**Review by Frenzie — APPROVED** [review]
Fine by me, waiting for @NiLuJe ^_^

**Review by NiLuJe — APPROVED** [review]

**Pull request merged**